go.php

```diff
@@ -11,6 +11,12 @@
     'admin-ldap'              => '/admin_manual/configuration/user_auth_ldap.html',
     'admin-provisioning-api'  => '/admin_manual/configuration/user_provisioning_api.html',
     'admin-source_install'    => '/admin_manual/installation/source_installation.html',
+    'admin-backup'            => '/admin_manual/maintenance/backup.html',
+    'admin-monitoring'        => '/admin_manual/operations/considerations_on_monitoring.html',
+    'admin-performance'       => '/admin_manual/configuration_server/performance_tuning.html',
+    'admin-config'            => '/admin_manual/configuration/configuration_config_sample_php.html',
```

Comment:
there is no configuration_config_sample_php in https://github.com/owncloud/documentation/tree/master/admin_manual